src/templates_parser-version.adb

```diff
@@ -0,0 +1,8 @@
+with GNAT.Bind_Environment;
+
+separate (Templates_Parser)
+function Version return String is
+   use GNAT;
+begin
+   return Bind_Environment.Get ("version");
+end Version;
```

Comment:
We need at least a CR our style check will choke on this. Can you fix ? Thanks.

Reply:
Why was I told then to convert to LF only?

Reply:
Sorry I'm confused I meant LF here.